mex/common/models/base.py

```diff
@@ -168,9 +166,42 @@ def fix_listyness(cls, data: _RawModelDataT) -> _RawModelDataT:
                     data[name] = cls._fix_value_listyness_for_field(field_name, value)
         return data
 
+    @model_validator(mode="wrap")
+    def verify_computed_field_consistency(
+        cls, data: Any, handler: ValidatorFunctionWrapHandler
+    ) -> Any:
+        """Validate that parsed values for computed fields are consistent.
+
+        Parsing a dictionary with a value for a computed field that is consistent with
+        what that field would have computed anyway is allowed. Omitting values for
+        computed fields is perfectly valid as well. However, if the parsed value is
+        different from the computed value, a validation error is raised.
+
+        Args:
+            data: Raw data or instance to be parsed
+            handler: Validator function wrap handler
+
+        Returns:
+            data with consistent computed fields.
+        """
+        if isinstance(data, MutableMapping):
```

Comment:
I am somewhat afraid that this if is skipped in yet unforeseen circumstances. That would lead to arbitrary values for computed fields being accepted and potentially different values returned when accessing the computed attributes. This could be covered by raising an Exception if the input type is unexpected. What do you think about this?

Reply:
hm, i explicitly didn't want to write custom error handling for cases where you try to parse invalid stuff like `BaseModel.model_validate("bla")` and rather have that be handled by pydantic

Reply:
i added a bailout for models without computed fields (so we don't unneccesarily restrict them from what they can parse) and an explicit error for non-dicts: https://github.com/robert-koch-institut/mex-common/pull/205/commits/8353d3c8ee827fd5eff14671b7ea5534355ec895

Reply:
Thank you!